code/game/mecha/combat/gorilla.dm

```diff
@@ -0,0 +1,219 @@
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
```

Comment:
These weapons are not intended for use on **any other mech, ever**. If this is even accepted, I'll move these into appropriate files if desired.